src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -96,46 +96,49 @@ export class TransactionApi implements ITransactionApi {
     await this.cacheService.deleteByKey(cacheKey);
   }
 
-  async getDataDecoded(data: string, to?: string): Promise<DataDecoded> {
+  async getDataDecoded(args: {
+    data: string;
+    to?: string;
+  }): Promise<DataDecoded> {
     try {
       const url = `${this.baseUrl}/api/v1/data-decoder/`;
       const { data: dataDecoded } = await this.networkService.post(url, {
-        data,
-        to,
+        data: args.data,
+        to: args.to,
       });
       return dataDecoded;
     } catch (error) {
       throw this.httpErrorFactory.from(error);
     }
   }
 
-  async getCollectibles(
-    safeAddress: string,
-    limit?: number,
-    offset?: number,
-    trusted?: boolean,
-    excludeSpam?: boolean,
-  ): Promise<Page<Collectible>> {
+  async getCollectibles(args: {
+    safeAddress: string;
+    limit?: number;
+    offset?: number;
+    trusted?: boolean;
+    excludeSpam?: boolean;
+  }): Promise<Page<Collectible>> {
     try {
       const cacheDir = CacheRouter.getCollectiblesCacheDir(
```

Comment:
All of the `CacheRouter` methods would also benefit from a similar change.

Reply:
[9db8374](https://github.com/safe-global/safe-client-gateway-nest/pull/610/commits/9db8374f600be9d8fbe58d5c91761aef0ea8b5cf)